metals/src/main/scala/scala/meta/internal/decorations/SyntheticsDecorationProvider.scala

```diff
@@ -488,7 +489,10 @@ final class SyntheticsDecorationProvider(
         case param: m.Term.Param =>
           if (param.decltpe.isEmpty) List(param.name.pos.toSemanticdb) else Nil
         case cs: m.Case =>
-          explorePatterns(List(cs.pat)) ++ visit(cs.body)
+          // if the case is too long then it'll be too messy
+          if (userConfig().showInferredType.contains("minimal"))
```

Comment:
We should probably also apply minimal for showing inferred type for methods. The case with `TypeApplyTree`

Reply:
Do you mean the type decoration for method invocation (where `foo()` becomes  `foo[t]()` ) ? I didn't find the relevant code

Reply:
It's here https://github.com/scalameta/metals/blob/main/metals/src/main/scala/scala/meta/internal/decorations/SemanticdbTreePrinter.scala#L223

Reply:
It seems to be modified before here: https://github.com/scalameta/metals/pull/5284/files#diff-ceb9daeb681f980f3b0d00a39f29318473295b32381925e64cd604a53b79715f

Reply:
Ach, I missed that, sorry! That's why it's probably good to add a test with the minimal configuration for types in mind

Reply:
cool, I will go on to add the test